builtin/game/statbars.lua

```diff
@@ -65,19 +67,26 @@ local function update_builtin_statbars(player)
 		hud.id_healthbar = nil
 	end
 
+	local breath     = player:get_breath()
 	local breath_max = player:get_properties().breath_max
 	if flags.breathbar and enable_damage and not immortal and
-			player:get_breath() < breath_max then
+			breath <= breath_max then
 		local number = 2 * scaleToDefault(player, "breath")
-		if hud.id_breathbar == nil then
- 			local hud_def = table.copy(breath_bar_definition)
+		if not hud.id_breathbar and breath < breath_max then
```

Comment:
Why `breath < breath_max`? Even if it's the same level I think the breath bar should be shown for a second.

Reply:
The breath bar would be flashing in and out of existence every time this function runs.